test/test_ot.py

```diff
@@ -118,6 +118,28 @@ def test_emd_empty():
     np.testing.assert_allclose(w, 0)
 
 
+def test_emd_sparse():
+
+    n = 100
+    rng = np.random.RandomState(0)
+
+    x = rng.randn(n, 2)
+    x2 = rng.randn(n, 2)
+
+    M = ot.dist(x, x2)
+
+    G = ot.emd([], [], M)
```

Comment:
I would explicitly pass `dense=True` for clarity (and in case that changes in the future)